src/opendr/perception/object_detection_2d/nms/seq2seq_nms/README.md

```diff
@@ -0,0 +1,13 @@
+Seq2Seq-NMS
+======
+
+This folder contains an implementation of Seq2Seq-NMS [[1]](#seq2seq_nms-1).
+
+TABLE-1: Average Precision (AP) achieved by pretrained models on the person detection task. The maximum number or RoIs, employed for the performance evaluation was set to 800.
+![Alt text](stats_pretrained.png?raw=true "Title")
```

Comment:
ditto, also there's extra spaces that can be removed

Reply:
I think its ok now